pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -208,6 +211,50 @@ func (buildSpec *BuildSpec) StrategyName() string {
 	return buildSpec.Strategy.Name
 }
 
+// VulnerabilityIgnoreOptions refers to ignore options for vulnerability scan
+type VulnerabilityIgnoreOptions struct {
+
+	// Issues references the security issues to be ignored in vulnerability scan
+	Issues []string `json:"issues,omitempty"`
+
+	// Severity indicates the severities of security issues to be ignored (comma separated)
+	Severity string `json:"severity,omitempty"`
+
+	// IgnoreUnfixed indicates flag to display only fixed vulnerabilities
+	Unfixed bool `json:"unfixed,omitempty"`
+}
+
+// VulnerabilityScanOptions references the options for vulnerability scanning
+type VulnerabilityScanOptions struct {
+
+	// Enabled indicates whether to run vulnerability scan for image
+	Enabled bool `json:"enabled,omitempty"`
+
+	// FailPush indicates whether to push the image if the vulnerability scan fails
+	FailPush bool `json:"fail,omitempty"`
+
+	// IgnoreOptions refers to ignore options for vulnerability scan
+	IgnoreOptions *VulnerabilityIgnoreOptions `json:"ignore,omitempty"`
```

Comment:
In light of https://github.com/shipwright-io/community/issues/186#issuecomment-1927111254, the JSON field name or variable name and type name should be revisited.

Reply:
Done